compile.sh

```diff
@@ -627,11 +628,29 @@ if [ "$COMPILE_GD" == "yes" ]; then
 	make install >> "$DIR/install.log" 2>&1
 	cd ..
 	echo " done!"
+	#libjpeg
+	echo -n "[libjpeg] downloading $LIBJPEG_VERSION..."
+	download_file "http://ijg.org/files/jpegsrc.v$LIBJPEG_VERSION.tar.gz" | tar -zx >> "$DIR/install.log" 2>&1
+	mv jpeg-$LIBJPEG_VERSION libjpeg
+	echo -n " checking..."
+	cd libjpeg
+	LDFLAGS="$LDFLAGS -L${DIR}/bin/php7/lib" CPPFLAGS="$CPPFLAGS -I${DIR}/bin/php7/include" RANLIB=$RANLIB ./configure \
```

Comment:
`CPPFLAGS`? shouldn't this be `CXXFLAGS`?

Reply:
why, CXXFLAGS would be for the C++ compiler. (And libjpeg is supposedly C?)

Reply:
then why is it CPPFLAGS and not CFLAGS?

Reply:
As far as I know CPPFLAGS point to the CPreProcessor, but I'm not very experienced/knowledgable at all here.